best-practices/folio_sdr_integrations.md

```diff
@@ -0,0 +1,108 @@
+# FOLIO and SDR Integrations
+
+## FOLIO Upgrades
+
+When Folio is upgraded, SDR integrations should be tested.  This is typically done as a FOLIO upgrade in -stage to allow for this testing.
+
+The following are some tests that can be done to verify the SDR integrations continue to work as expected.  It is not necessarily exhaustive, but will cover the major touch points.
+
+Note: it is possible that the data in Folio stage may not match the data in Argo stage, which will lead to barcodes/instance HRIDs in argo stage that are not found in Folio stage.  This may be because the data in Folio stage was replaced with a snapshot of Folio prod data.  This may mean you will need to ask for help in #sdr-operations or #libsys-infra-folio-integration to find example objects to work with (or trying looking in Argo prod for relevant objects).
+
+### Integration Tests
+
+Run the following infrastructure integration tests and ensure they pass:
+
+- spec/features/item_creation_with_folio_hrid_spec.rb
+- spec/features/sdr_deposit_spec.rb
+- spec/features/etd_creation_spec.rb
+
+### Searchworks Release
+
+1. Find an item in Argo stage with a Folio Instance HRID, so that it can be released.  Here is one example: https://argo-stage.stanford.edu/view/druid:bc930hq4019, ask in #sdr-operations or #libsys-infra-folio-integration for others).
```

Comment:
```suggestion
1. Find an item in Argo stage with a Folio Instance HRID, so that it can be released.  Here is one example: https://argo-stage.stanford.edu/view/druid:bc930hq4019, ask in #sdr-operations or #libsys-infra-folio-integration for others.
```